lightningd/entrypoint-shim.sh

```diff
@@ -8,5 +8,12 @@ if [ -n "$LIGHTNINGD_REPLICA" ]; then
 fi
 
 jinja2 /config.j2 > "${LIGHTNINGD_DATA}/config"
+if [ -d ${LIGHTNINGD_DATA}/plugins ]
```

Comment:
probably simpler just to remove this bit and modify the dockerfile from:

`COPY ./rebalance.py /opt/lightningd_plugins/`
to
`COPY ./rebalance.py $LIGHTNINGD_DATA/plugins`

Reply:
im sure i've tried this awhile back. but didn't work. its like the config situation where its either empty, not being created or its being forgotten. either way the file being COPY over doesnt exists when the container starts.

Reply:
yep just confirmed it... its looking like adding a file using COPY from dockerfile doesn't copy the file to the ${LIGHTNINGD_DATA}/plugins directory. even when the plugins directory exists(which doesn't appear to be the case initially).

Reply:
ok I figured out why we cant modify $LIGHTNINGD_DATA in the dockerfile and documented it